src/main/java/org/sunbird/digilocker/model/PullURIResponse.java

```diff
@@ -0,0 +1,184 @@
+package org.sunbird.digilocker.model;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+import javax.xml.bind.annotation.XmlElement;
+
+@JacksonXmlRootElement(localName = "PullURIResponse", namespace = "http://tempuri.org/")
+public class PullURIResponse {
+
+    private ResponseStatus responseStatus;
+    private DocDetails docDetails;
+
+    public PullURIResponse() {
+        this.responseStatus = new ResponseStatus();
+        this.docDetails = new DocDetails();
+    }
+
+    @JacksonXmlProperty(localName = "ResponseStatus", namespace = "http://tempuri.org/")
+    public ResponseStatus getResponseStatus() {
+        return responseStatus;
+    }
+
+    public void setResponseStatus(ResponseStatus responseStatus) {
+        this.responseStatus = responseStatus;
+    }
+
+    @JacksonXmlProperty(localName = "DocDetails", namespace = "http://tempuri.org/")
+    public DocDetails getDocDetails() {
+        return docDetails;
+    }
+
+    public void setDocDetails(DocDetails docDetails) {
+        this.docDetails = docDetails;
+    }
+
+    // Inner class for ResponseStatus
+    public static class ResponseStatus {
```

Comment:
I use inner class because of mapping the xml format i can create different class but their are 6 object which i check for which need to create sperate class

Reply:
Take it as ResponseStatus only and used on both pullURI and pullDoc Request